Packs/Base/Scripts/StixParser/StixParser.py

```diff
@@ -1221,6 +1221,21 @@ def decode(props, **kwargs):
         return domains
 
 
+def detect_indicator_type_file(indicator_value: str):
```

Comment:
don't we have something similar in common server python?

Reply:
Nope

Reply:
looks that it can be useful to write this one in CommonServerPython maybe

Reply:
It does, I also thought we have it (I even remember writing something like it somewhere), but it should be as a different PR as CSP is the worst.